pkg/cloudvolumes/login.go

```diff
@@ -0,0 +1,128 @@
+// (C) Copyright 2019 Hewlett Packard Enterprise Development LP.
+
+package cloudvolumes
+
+import (
+	"fmt"
+	"github.com/howeyc/gopass"
+	"github.com/HewlettPackard/hpecli/pkg/logger"
+	"github.com/HewlettPackard/hpecli/pkg/store"
+	"github.com/spf13/cobra"
+	"encoding/json"
+	"net/http"
+	"io/ioutil"
+	"bytes"
+	"crypto/tls"
+)
+
+var (
+	host     string
+	username string
+	password string
+	token	 string
+)
+
+func init() {
+	cmdCloudVolumesLogin.Flags().StringVar(&host, "host", "", "Cloud Volumes portal")
+	cmdCloudVolumesLogin.Flags().StringVarP(&username, "username", "u", "", "ilo username")
+	cmdCloudVolumesLogin.Flags().StringVarP(&password, "password", "p", "", "ilo passowrd")
+}
+
+// getCmd represents the get command
+var cmdCloudVolumesLogin = &cobra.Command{
+	Use:   "login",
+	Short: "Login to Cloud Volumes: hpecli cloudvolumes login",
+	RunE:  runLogin,
+}
+
+func runLogin(cmd *cobra.Command, args []string) error {
+	fmt.Println("cloudvolumes/login called")
+
+	if host == "" {
+		return fmt.Errorf("must provide --host or -h")
+	}
+	//if !strings.HasPrefix(host, "http") {
+	//	host = fmt.Sprintf("http://%s", host)
+	//}
+
+	if username == "" {
+		// return fmt.Errorf("must provide --username or -u")
+		fmt.Print("username: ")
+		//var input string
+		fmt.Scanln(&username)
+		//fmt.Print(input)
+	}
+
+	if password == "" {
+		// this really isn't secure to provide on the command line
+		// need to provide a way to read from stdin
+		//return fmt.Errorf("must provide --password or -p")
+		fmt.Print("password: ")
+		pass, _ := gopass.GetPasswd()
```

Comment:
I believe this is was from when terminal didn't exist and have the ability to do something similar...
Take a look at:
https://godoc.org/golang.org/x/crypto/ssh/terminal
Consider using it instead of gopass